iati_datastore/iatilib/console.py

```diff
@@ -75,30 +75,6 @@ def build_query_builder(deploy_url=None):
         subprocess.run(['npm', 'run', 'generate'], cwd=cwd)
 
 
-@click.option(
-        '-x', '--fail-on-xml-errors', "fail_xml")
-@click.option(
-        '-s', '--fail-on-spec-errors', "fail_spec")
-@click.option('-v', '--verbose', "verbose")
-@click.argument('filenames', nargs=-1)
-@cli.command()
-@with_appcontext
-def parse_file(filenames, verbose=False, fail_xml=False, fail_spec=False):
```

Comment:
Are we sure this isn't useful? I quite like having command line commands to quickly test particular files etc?

Reply:
Yeah, fair enough. I am not really sure if it is useful tbh. Let’s keep it!